roles/lnls-ans-role-sirius-apps/vars/Debian-buster.yml

```diff
@@ -1,11 +1,14 @@
 ---
 __sirius_apps_epics_packages:
-  - xpra=2.4.3*
-  # - winswitch
-  - xvfb=2:1.20.4*
-  - bc=1.07.1*
+  - "epics-extensions-dev={{ pkg_version_epics_extensions_dev }}"
+  - "edm-dev={{ pkg_version_edm_dev }}"
+  - "xpra={{ pkg_version_xpra }}"
+  - "winswitch={{ pkg_version_winswitch }}"
```

Comment:
Small detail. There is no winswitch package for Debian Buster. We should comment it (as a means to inform the developer that wants to use the role with Buster to search for a replacement) or remove it completely in my opinion

Reply:
ok @lerwys!, I will do it!

Reply:
done!